metals/src/main/scala/scala/meta/metals/Configuration.scala

```diff
@@ -27,16 +27,17 @@ object Configuration {
 
   @JsonCodec case class Sbt(
       enabled: Boolean = false,
-      command: String = "test:compile"
+      diagnostics: Diagnostics = Diagnostics(enabled = true),
+      command: String = "test:compile",
   )
   @JsonCodec case class Scalac(
-      completions: ScalacCompletions = ScalacCompletions(),
-      diagnostics: ScalacDiagnostics = ScalacDiagnostics(),
+      completions: Completions = Completions(enabled = false),
+      diagnostics: Diagnostics = Diagnostics(enabled = false),
   ) {
     def enabled: Boolean = completions.enabled || diagnostics.enabled
   }
-  @JsonCodec case class ScalacCompletions(enabled: Boolean = false)
-  @JsonCodec case class ScalacDiagnostics(enabled: Boolean = false)
+  @JsonCodec case class Completions(enabled: Boolean = false)
+  @JsonCodec case class Diagnostics(enabled: Boolean = false)
```

Comment:
Since these two classes are now shared, I would remove the default value from here.

Reply:
Unrelated, I've wondered if we should have `case class Enabled(enabled: Boolean)` and use it like `completions: Enabled`

Reply:
Yep, that crossed my mind too. My reason for initially using different classes is that I assumed they would diverge. Since it's a cheap change, we could probably get away with a single `Enabled` class now and diverge later if necessary.

Reply:
Done